ksql-benchmark/src/main/java/io/confluent/ksql/benchmark/SerdeBenchmark.java

```diff
@@ -192,20 +192,26 @@ public void setUp(final SchemaAndGenericRowState rowState) {
     }
   }
 
+  @SuppressWarnings("MethodMayBeStatic") // Tests can not be static
   @Benchmark
   public byte[] serialize(final SerdeState serdeState) {
     return serdeState.serializer.serialize(TOPIC_NAME, serdeState.row);
   }
 
+  @SuppressWarnings("MethodMayBeStatic") // Tests can not be static
   @Benchmark
   public GenericRow deserialize(final SerdeState serdeState) {
     return serdeState.deserializer.deserialize(TOPIC_NAME, serdeState.bytes);
   }
 
-  public static void main(final String[] args) throws RunnerException {
-    final Options opt = new OptionsBuilder()
-        .include(SerdeBenchmark.class.getSimpleName())
-        .build();
+  public static void main(final String[] args) throws Exception {
+
+    final Options opt = args.length != 0
+        ? new CommandLineOptions(args)
+        : new OptionsBuilder()
+            .include(SerdeBenchmark.class.getSimpleName())
+            .shouldFailOnError(true)
```

Comment:
What is this argument that controls whether to fail on error? Is this documented somewhere?

Reply:
Not sure where it's documented. Just means if the code under test throws exceptions it fails, which is what we want.